File: official/docs/csharp/v3/referral/create-referral-user.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using Newtonsoft.Json;
using EasyPost;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            EasyPost.ClientManager.SetCurrent("EASYPOST_API_KEY");

            ReferralCustomer referralUser = await Partner.CreateReferral(
                new Dictionary<string, object>
                {
                    { "name", "test user" },
                    { "email", "email@example.com" },
                    { "phone", "8888888888" }
                }
            );

            Console.WriteLine(JsonConvert.SerializeObject(referralUser, Formatting.Indented));
        }
    }
}

File: official/guides/partner-white-label-guide/golang/create-referral-user.go
package example

import (
    "fmt"
    "os"

    "github.com/EasyPost/easypost-go/v2"
)

func main() {
    apiKey := os.Getenv("<YOUR_TEST/PRODUCTION_API_KEY>")
    client := easypost.New(apiKey)

    name := "Test Referral"
    email := "test@example.com"
    phone := "5555555555"

    referralUser, _ := client.CreateReferralCustomer(
        &easypost.UserOptions{
            Name:  &name,
            Email: &email,
            Phone: &phone,
        },
    )

    fmt.Println(referralUser)
}
